src/clients/ETHClient.ts

```diff
@@ -126,6 +135,58 @@ export class ETHClient {
     return new BigNumber(allowance.toString())
   }
 
+  public async bridgeTokens(params: BridgeParams): Promise<EthersTransactionResponse> {
+    const { fromToken, toToken, amount, recoveryAddress, ethAddress, signer, gasPriceGwei, gasLimit } = params;
+
+    const networkConfig = this.getNetworkConfig();
+    const rpcProvider = this.getProvider();
+    const contractAddress = this.getLockProxyAddress();
+    
+    const fromTokenId = fromToken.id;
+    const fromTokenAddress = appendHexPrefix(fromToken.tokenAddress);
+    const toTokenDenom = toToken.denom;
```

Comment:
seems wrong, should be using id if im not wrong, can you test on the tokens where `id` and `denom` are different?